docs/jwst/dark_current/arguments.rst

```diff
@@ -1,10 +1,15 @@
 Step Arguments
 ==============
 
-The dark current step has one step-specific argument:
+The dark current step has these step-specific arguments:
 
 *  ``--dark_output``
 
 If the ``dark_output`` argument is given with a filename for its value,
 the frame-averaged dark data that are created within the step will be
 saved to that file.
+
+*  ``--average_dark_current``: The average dark current for this detector in units of electrons
+  per second. This will be used to calculate the Poisson noise contribution due to the dark
```

Comment:
Sphinx build in the CI tests is failing at this line, because the indent level of lines 13-15 is not what's expected. I think these lines need to start in column 1, like lines 8-10 above.